client/src/pages/Register.tsx

```diff
@@ -31,14 +31,36 @@ const useStyles = makeStyles((theme) => ({
 			margin: theme.spacing(2),
 		},
 	},
+	margin: {
+		marginTop: theme.spacing(4),
+	},
 	alert: {
 		marginTop: '20px',
 	},
+	buttonSpan: {
+		textDecoration: 'underline',
+		cursor: 'pointer',
+		color: '#069',
+		background: 'none',
+		border: 'none',
+	},
+	label: {
+		marginBottom: theme.spacing(4),
+	},
+	roundButton: {
+		borderRadius: '20px',
+		width: '300px',
+		fontWeight: 700,
+	},
+	input: {
+		minWidth: '300px',
+		margin: theme.spacing(2),
+	},
 }));
 
 const emptyErrors = { username: [], email: [], password: [], confirmPassword: [], server: [] };
 
-export const Register = () => {
+export const Register = ({ handleFormChange }) => {
 	const { login } = useContext(AuthContext);
 
 	const [formValues, setFormValues] = useState({
```

Comment:
This whole logic ( from line 66 to 115) can be easily replaced by logic that we already have inside **ContainerForLandingPage**. Existing logic is now used for **Login** component. In this case, it can be easily adjusted.